app/signup.tsx

```diff
@@ -0,0 +1,275 @@
+import React, { useState } from 'react';
+import {
+  View,
+  Text,
+  StyleSheet,
+  SafeAreaView,
+  TextInput,
+  TouchableOpacity,
+  KeyboardAvoidingView,
+  Platform,
+  TouchableWithoutFeedback,
+  Keyboard
+} from 'react-native';
+import { Ionicons } from '@expo/vector-icons';
+import { router } from 'expo-router';
+
+export default function SignUpScreen() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+  const [showPassword, setShowPassword] = useState(false);
+  const [showConfirmPassword, setShowConfirmPassword] = useState(false);
+  const [isEmailFocused, setIsEmailFocused] = useState(false);
+  const [isPasswordFocused, setIsPasswordFocused] = useState(false);
+  const [isConfirmPasswordFocused, setIsConfirmPasswordFocused] = useState(false);
+
+  const handleSignUp = () => {
+    // Implement signup logic here
+    console.log('Sign up with:', email, password);
+    // On successful signup:
+    router.replace('/(tabs)');
+  };
+
+  return (
+    <TouchableWithoutFeedback onPress={Keyboard.dismiss}>
```

Comment:
Same issue as `login.tsx`, I am unable to open the keyboard for any of the inputs.